internal/pkg/scorecard/scorecard.go

```diff
@@ -475,7 +189,7 @@ func initConfig() error {
 
 func configureLogger() error {
 	if viper.GetString(OutputFormatOpt) == HumanReadableOutputFormat {
-		logReadWriter = os.Stdout
+		logReadWriter = os.Stderr
```

Comment:
What's the reason for this change?

Reply:
This was a leftover change from when I was splitting the plugins from the sdk binary. This should be Stdout.

Fixed